### DISCUSSION TITLE
🐙 Thank You for Your Support and an Update on Our Progress

### DISCUSSION THREAD

**Discussion by Leo310** [nicobrauchtgit/obsidian-Smart2Brain#149] [Announcements]
Dear S2B community,

First and foremost, thanks for all the bug reports, feature requests, and overall valuable feedback 🙏❤️. 

Second, we want to apologize for our recent absence. Over the past few weeks, our focus has been primarily on writing our bachelor theses, which we are scheduled to submit in August. This demanding task has required most of our attention and time, limiting our ability to engage with the community and work on the S2B plugin as much as we would have liked. 

The good news is that both of our theses are centered around Retrieval-Augmented Generation (RAG) optimizations. This research is highly relevant to the future development of the S2B plugin, and we are eager to apply the theoretical knowledge and insights we have gained during this period.

We are excited about the future and the improvements we will bring to the S2B plugin post-submission. Your ongoing support and patience are greatly appreciated, and we look forward to re-engaging with the community and implementing the enhancements that will make the Smart Second Brain even better.

Thank you once again for your support and contributions and stay tuned for more updates!

Your S2B team,
Nico, Basti, and Leo

**Comment by lukmay** [comment]
Good luck with your bachelor's thesis! 
You all rock, and I'm sure you'll do great!

I'm very excited to see your research included in the S2B project.🔆

**Comment by fdominguezr** [comment]
Congratulatoins for S2B and I hope you've got the best grades possible with your bachelor theses. I think you all deserve it.
Furthermore, after testing your tool, I have a long hanging fruit, that can improve dramatically both user experience and then, adoption. See it here: [Models' self suggestion capability](https://github.com/your-papa/obsidian-Smart2Brain/issues/162)

Anyway, I wish you all the best and keep going!
Thanks!

**Comment by grayerbeard** [comment]
So now its nearly year's end what is the status of this project now?

**Reply by TomLucidor** [reply]
Seconding this but 3 months later...

**Reply by aalencia** [reply]
and again?

**Reply by Leo310** [reply]
I know its been some time so I want to apologize again. Life has been busy and technology is progressing faster than one can think. 

After a break in 2024/25, we continued the development and have been working on the project in the background for the past months.
With that said, I can proudly share that S2B V2 will be released in the next few weeks! 
Feel free to try out the [beta](https://github.com/your-papa/obsidian-Smart2Brain/releases/tag/2.1.0-beta) until then (you can install it with [obsidian-brat](https://community.obsidian.md/plugins/obsidian42-brat)).

Thanks for your patience! 😅

**Reply by TomLucidor** [reply]
Please take as much time as needed for `sota-scan` so that it can have parity to other Obsidian AI apps, supporting multiple inference engines of small models (including `rapid-mlx` and other high-speed engines) as well as diverse APIs. Also self-improvement/correction/generalization + vector search (for implicit topic clustering) with graph search (for argumentation or context or derivation) combos + skill "borg" for diverse tools would be great!

**Comment by Lasklu** [comment]
Congratulations! Can you share details on some upcoming features? So excited! xoxo

**Reply by Leo310** [reply]
Thanks, S2B V2 will be released in the next few weeks!
Feel free to try out the [beta](https://github.com/your-papa/obsidian-Smart2Brain/releases/tag/2.1.0-beta) until then (you can install it with [obsidian-brat](https://community.obsidian.md/plugins/obsidian42-brat)).
